rrfs-test/CMakeLists.txt

```diff
@@ -2,6 +2,22 @@
 #!    NOAA/NCEP, National Centers for Environmental Prediction  RDASapp        !
 #!-------------------------------------------------------------------------
 
+include(funcs.cmake)
+
+### Define CTests here ###
+
+# FV3-JEDI tests
+set(rrfs_fv3jedi_tests "")
```

Comment:
#dic1
set (rrfs_tests_execs)
add_to_dictionary(rrfs_tests_execs  "rrfs_fv3jedi_hyb_2022052619"  "fv3jedi_var.x") 

and all the similar to the same. 
In fact, to add corresponding "add_to_dictionary(rrfs_tests_execs, <test_name>, <test_exec>) " should be done by the developer who is adding a new test